examples/cpp/customserializer/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Leave customer serializer example out of this pull and ticket.

Reply:
will do.